### PR TITLE
[Quikfix] disable ugly outline

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/assets/stylesheets/base.scss [hunk 1]
```diff
@@ -3,6 +3,8 @@ body {
   padding: none;
 }
 
+a:focus, a:active, button:focus, button:active { outline: none; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/base.scss]: Each selector in a comma sequence should be on its own single line

File: app/assets/stylesheets/base.scss [hunk 2]
```diff
@@ -3,6 +3,8 @@ body {
...
+
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/base.scss]: Each selector in a comma sequence should be on its own single line

File: app/assets/stylesheets/mobile/mobile.scss [hunk 3]
```diff
@@ -21,6 +21,8 @@ a {
   text-decoration: none;
 }
 
+a:focus, a:active, button:focus, button:active { outline: none; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/mobile/mobile.scss]: Each selector in a comma sequence should be on its own single line

File: app/assets/stylesheets/mobile/mobile.scss [hunk 4]
```diff
@@ -21,6 +21,8 @@ a {
...
+
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/mobile/mobile.scss]: Each selector in a comma sequence should be on its own single line

**Comment by svbergerem** [comment]
This “ugly outline” tells users on most websites (including github, youtube, google, facebook and others) which link is focused.

http://outlinenone.com

**Comment by jhass** [comment]
Agreed, this won't go in.

**Comment by AugierLe42e** [comment]
Ok.

**Pull request closed without merging**